### ISSUE TITLE
Fix "import" for miniRT modules

### ISSUE THREAD

**Issue by fatcerberus** [spheredev/neosphere#159]
Unlike Babel, TypeScript doesn't attempt to do anything to make `import` and `require()` interoperate.  As a result, all miniRT modules must be imported like this:
```js
import * as term from 'term';
term.print("...");
```

For modules such as "prim", that style of import makes sense, since such modules are logically collections of independent functions.  But it makes no sense for "term" or "from", which export only a single object by design:

```js
import * as from from 'from';

// the import syntax used suggests 'from' is a collection of named exports, but it's really
// just a solitary function:
from([1,2,3]).each(v => console.log(v));
```

I could return to using Babel as the transpiler, but there's a good chance they will do away with the CommonJS/ESM interop eventually as well, leaving me in the same predicament.  However, improving the compatibility for `import` will be a breaking change for existing users of `require()`:

```js
const from = require('from').default;
```

Not very pretty.  This will need some thought.

**Comment by fatcerberus** [comment]
See https://github.com/Microsoft/TypeScript/issues/16093.  TypeScript is eventually going to align with the Babel behavior, but that will take some time.  I would prefer not to wait several months for that fix, so something is going to have to be done on our end.

**Comment by fatcerberus** [comment]
This is a showstopper for miniSphere 4.6 and needs some kind of fix now, even if it's just a band-aid fix for now.  I'll see what I can do in the next day or so.

**Comment by fatcerberus** [comment]
For miniSphere 4.6 I don't want to cause major breakage, so I'll just do something like this to get `import` to cooperate:

```js
module.exports.default = module.exports
```

For miniSphere 5.0 I'll completely refactor the modules to optimize for ES6, since that's the preferred development platform for Sphere v2 anyway.  That probably means more `require('module').default` for non-ES6 code, but if that's the price to be paid, so be it.

**Comment by fatcerberus** [comment]
So much for "no major breakage".  Rather than implement a hack to fix this now just to have to change it again, I decided to do what I wanted to do later, now: I promoted miniRT to an official part of the Sphere v2 standard library, with an API to match.

The CommonJS library is split into individual modules, one for each object.  For example `Scene` is imported as `require('scene')` or `from()` as `require('from')`.  There are no modules that export more than one object, so that you'd never have to something like `require('module').Thing`.

The ES6 library, by contrast, is combined into a single module, `sphere-runtime`, to facilitate importing multiple objects at once.  The same objects are exposed as the CommonJS library, so that one would put at the top of a module something like:
```js
import { from, Music, Scene, Thread } from `sphere-runtime`;
```

The reorganization described above is implemented in master, so this can be considered fixed. 😸
